task_2/7.js

```diff
@@ -0,0 +1,21 @@
+// Создать пустой объект и наполнить его данными:
+// 1. Добавить свойство name и записать в него значение ‘Alex’
+// 2. Добавить свойство lastName и записать в него значение ‘Smith’
+// 3. Присвоить значение ‘Bob’ в свойство name
+// 4. Удалить оба свойства
+
+const obj = Object.create({}, {
+  name: {
+    value: 'Alex',
+    writable: true,
+    configurable: true
+  },
+  lastName: {
+    value: 'Smith',
+    configurable: true
+  }
+})
```

Comment:
Полагаю, что можно сократить объявление нового объекта, если нет необходимости в назначении специальных атрибутов для его свойств
```suggestion
const obj = {
  name: 'Alex',
  lastName: 'Smith'
}
```